cli/config/config.go

```diff
@@ -0,0 +1,13 @@
+package config
```

Comment:
Yes, this can get bucketed into the same category as the comment above it (Will probably need to happen eventually but can be deferred for now).